src/Ilios/CoreBundle/Classes/UserMaterial.php

```diff
@@ -39,6 +44,20 @@ class UserMaterial
      * @IS\Expose
      * @IS\Type("string")
      */
+    public $clm;
```

Comment:
I think there is some value in expanding this property to be `courseLearningMaterial` for long term clarity.